Server/Plugins/APIDump/main_APIDump.lua

```diff
@@ -136,7 +136,7 @@ local function CreateAPITables()
 		return res;
 	end
 
-	for i, v in pairs(_G) do
+	for i, v in pairs(getmetatable(_G).__index) do
 		if (
 			(v ~= _G) and           -- don't want the global namespace
```

Comment:
Will this comparison still work? I doubt that.

Reply:
It actually does still work.

The `_preload` code doesn't replace the actual `_G` table. It just takes its values and moves it to a new table. So once it gets to the `_G` key it takes the reference to itself and moves it to the new table.

Put in another way: Usually `_G` references to itself like this:
```
_G._G._G._G...
```

but with `_preload` this changes to
```
_G.__index._G.__index...
  ▲
   because of getmetatable() we start here which means its _G value 
   references back to the global _G environment
```